airflow/providers/jenkins/operators/jenkins_job_trigger.py

```diff
@@ -235,3 +238,4 @@ def execute(self, context):
             # If we can we return the url of the job
             # for later use (like retrieving an artifact)
             return build_info['url']
+        return None
```

Comment:
It kinda made me add this after complaining about no return statement. It wouldn't even let me use just `return`. Is this a known issue/attribute of `mypy`?

Reply:
This is mypa's expected behavior. If the function exits without calling `return`, Python implicitly returns None. Mypy requires us to be implicit in this case.

 > Explicit is better than implicit.

https://www.python.org/dev/peps/pep-0020/

Reply:
Got it, that makes sense :+1: